/jet/home/palavall/.conda/envs/mpalaval-research/lib/python3.10/site-packages/huggingface_hub/file_download.py:980: UserWarning: Not enough free disk space to download the file. The expected file size is: 6431.83 MB. The target location /jet/home/palavall/.cache/huggingface/hub only has 0.00 MB free disk space.
  warnings.warn(
/jet/home/palavall/.conda/envs/mpalaval-research/lib/python3.10/site-packages/huggingface_hub/file_download.py:980: UserWarning: Not enough free disk space to download the file. The expected file size is: 6431.83 MB. The target location /jet/home/palavall/.cache/huggingface/hub/models--gpt2-xl/blobs only has 0.00 MB free disk space.
  warnings.warn(
Downloading model.safetensors:   0%|          | 0.00/6.43G [00:00<?, ?B/s]Downloading model.safetensors:   0%|          | 21.0M/6.43G [00:00<00:48, 132MB/s]Traceback (most recent call last):
  File "/ocean/projects/cis230002p/palavall/nanoGPT/train.py", line 182, in <module>
    model = GPT.from_pretrained(init_from, override_args)
  File "/ocean/projects/cis230002p/palavall/nanoGPT/model.py", line 238, in from_pretrained
    model_hf = GPT2LMHeadModel.from_pretrained(model_type)
  File "/jet/home/palavall/.conda/envs/mpalaval-research/lib/python3.10/site-packages/transformers/modeling_utils.py", line 2778, in from_pretrained
    resolved_archive_file = cached_file(pretrained_model_name_or_path, filename, **cached_file_kwargs)
  File "/jet/home/palavall/.conda/envs/mpalaval-research/lib/python3.10/site-packages/transformers/utils/hub.py", line 429, in cached_file
    resolved_file = hf_hub_download(
  File "/jet/home/palavall/.conda/envs/mpalaval-research/lib/python3.10/site-packages/huggingface_hub/utils/_validators.py", line 118, in _inner_fn
    return fn(*args, **kwargs)
  File "/jet/home/palavall/.conda/envs/mpalaval-research/lib/python3.10/site-packages/huggingface_hub/file_download.py", line 1429, in hf_hub_download
    http_get(
  File "/jet/home/palavall/.conda/envs/mpalaval-research/lib/python3.10/site-packages/huggingface_hub/file_download.py", line 554, in http_get
    temp_file.write(chunk)
  File "/jet/home/palavall/.conda/envs/mpalaval-research/lib/python3.10/tempfile.py", line 483, in func_wrapper
    return func(*args, **kwargs)
OSError: [Errno 122] Disk quota exceeded
Downloading model.safetensors:   0%|          | 31.5M/6.43G [00:00<01:07, 95.4MB/s]
